modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/IgniteJraftServiceFactory.java

```diff
@@ -38,7 +39,9 @@ public class IgniteJraftServiceFactory extends DefaultJRaftServiceFactory {
     private final LogStorageFactory logStorageFactory;
 
     /** Snapshot storage factory. */
-    private volatile SnapshotStorageFactory snapshotStorageFactory = LocalSnapshotStorage::new;
+    private volatile SnapshotStorageFactory snapshotStorageFactory = (path, raftOptions, logManager) -> {
+        return new LocalSnapshotStorage(path, raftOptions);
+    };
```

Comment:
```suggestion
    private volatile SnapshotStorageFactory snapshotStorageFactory = (path, raftOptions, logManager) -> new LocalSnapshotStorage(path, raftOptions);
```

Reply:
This way the line will exceed 140 characters in length

Reply:
You can add line breaks.

Reply:
Well, I kinda did :)

Reply:
Not quite.